src/components/World/AddNodeDialog.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from "react";
```

Comment:
### Can't use dropdowns when objects are moving

This is a bit bizarre but... when objects are in motion, all dropdowns stop working. You can expand the dropdown, but you can't select an option.

Easiest way to repro is to add an object at (0,0,0) so it starts under the floor and falls forever. Then add another object and try to use the dropdowns.

Reply:
Fixed via PR